src/machines/commerce-machine.ts

```diff
@@ -12,6 +12,36 @@ const findAvailablePPPCoupon = (pricingData: PricingData) => {
   return pricingData?.available_coupons?.ppp
 }
 
+const extractAppliedDefaultCoupon = (
```

Comment:
Nothing new about this function, just moved it up here so that it can be called from multiple actions.